pkg/util/metric/m_process.go

```diff
@@ -141,3 +142,57 @@ func (c procFdsLimit) Metric(s *statCaches) (prom.Metric, error) {
 		return nil, moerr.NewInternalError("empty limit")
 	}
 }
+
+var ProcConnections = newBatchMetricVec(newProcConnections())
```

Comment:
please consider using NewGaugeVec with an account label to record the connection count